css/style.css

```diff
@@ -1,4 +1,330 @@
+}
+
+.content__data-car {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 256px 22px 270px;
+}
+
+.content__facts {
+  display: flex;
+  margin-right: 81px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 3px 0 0 3px;
+}
+
+.data__speed-icon {
+  width: 54px;
+  height: 42px;
+  padding-right: 10px;
+}
+
+.content__road-icon {
+  width: 40px;
+  height: 59px;
+  padding-right: 7px;
+  padding-top: 7px;
+}
+
+.data__speed-block {
+  padding-right: 40px;
+}
+
+.data__road-block {
+padding-bottom: 4px;
+}
+
+.data__figures {
+  font-weight: 600;
+  font-size: 35px;
+  padding-right: 3px;
+}
+
+.data__speed {
+  font-weight: 600;
+  font-size: 21px;
+}
+
+.data__specifications {
+  display: inline-block;
+  font-size: 13px;
+}
+
+.test-btn {
+  display: flex;
+  padding-right: 19px;
+}
+
+.testdrive {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  white-space: nowrap;
+  width: 118px;
+  margin-right: 40px;
+}
+
+.testdrive__img {
+  width: 22px;
+  height: 22px;
+  padding-right: 6px;
+}
+
+.testdrive__text {
+  font-weight: bold;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.order-btn {
+  border: 2px solid #e51937;
+  width: 160px;
+  height: 40px;
+  background-color: #fff;
+  font-weight: bold;
+  font-size: 14px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
+  transition: all 0.5s ease 0s; 
+}
+
+.order-btn:hover {
+  cursor: pointer;
+  border: none;
+  color: #fff;
+  background-color: #e51937;
+  transition: all 0.5s ease 0s; 
+  box-shadow: 0px 3px 3px 1px #9b9999;
+}
+
+.arrow {
+  background-color: #fff;
+  width: 24px;
+  height: 15px;
+  position: relative;
+  transition: all 0.5s ease 0s;
+  margin: auto;
+}
+
+.arrow__line {
+  position: absolute;
+  width: 22px;
+  height: 3px;
+  background-color: #e51937;
+}
+
+.first-ln {
+  transform: translate(-7px, 7px) rotate(45deg);
+  top: 7px; 
+}
+
+.second-ln {
+  transform: translate(7px, -7px) rotate(-45deg); 
+  top: 21px;
+}
+
+.arrow:hover {
+  cursor: pointer;
+}
+
+@media (min-width:1024px) and (max-width:1280px) {
+  .menubar__link:nth-child(5) {
+    margin-right: 170px;
+  }
+  .content {
+    background-size: contain;
+    background-position-y: 0px;
+    background-position-x: 30px;
+  }
+  .content__title {
+    margin-bottom: 390px;
+  }
+  
+}
+
+@media (min-width:768px) and (max-width:1024px) {
+  .header {
+    padding: 25px 24px 98px 0;
+  }
+  .header__logo-img{
+    padding-left: 16px;
+  }
+  .menubar__link {
+    display: none;
+  }
+  .button {
+    width: 33px;
+    height: 20px;
+  }
+  .second {
+    top: 9px;
+  }
+  .third {
+    top: 19px;
+  }
+  .content {
+    background-size: cover;
+    background-position-x: 20px;
+    background-position-y: 90px;
+  }
+  .content__title {
+    margin-bottom: 390px;
+  }
+  .content__data-car {
+    flex-direction: column;
+  }
+}
+ @media (min-width:568) and (max-width: 768px) {
+
+ }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)